contracts/token/ERC721/utils/ERC721Holder.sol

```diff
@@ -22,7 +22,7 @@ contract ERC721Holder is IERC721Receiver {
         address,
         uint256,
         bytes memory
-    ) public virtual override returns (bytes4) {
```

Comment:
This is unrelated with ERC20, and would break the overridability of the function.